packages/tasks/src/tasks/audio-text-to-text/about.md

```diff
@@ -0,0 +1,184 @@
+## Use Cases
+
+> This task takes **audio** and an optional **text prompt** and returns **text** (answers, summaries, structured notes, etc.).
+
+### Audio question answering
+Ask targeted questions about lectures, podcasts, or calls and get prices, and context-aware answers.
+**Example:** Audio: physics lecture → Prompt: “What did the teacher say about gravity and how is it measured?”
+
+### Meeting notes & action items
+Turn multi-speaker meetings into concise minutes with decisions, owners, and deadlines.
+**Example:** Audio: weekly stand-up → Prompt: “Summarize key decisions and list action items with assignees.”
+
+### Speech understanding & intent
+Go beyond transcription to extract intent, sentiment, uncertainty, or emotion from spoken language.
+**Example:** “I’m not sure I can finish this on time.” → Prompt: “Describe speaker intent and confidence.”
+
+### Lecture/podcast summarization with follow-ups
+Produce a brief summary, then answer follow-up questions that reference the audio.
+**Example:** Audio: 30-min podcast → Prompt: “Give a 5-bullet summary, then explain the guest’s main argument.”
+
```

Comment:
```suggestion

```

Let's remove this since it's quite similar to second point with `Meeting notes & action items` - we can update that with details of this point in a concise manner.